contributing/consul-mermaid-theme.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this meant to be included in the PR?

Reply:
Ah yes, but I should probably include some details about it in the README.

This can be added as the config to https://mermaid-js.github.io/mermaid-live-editor/edit to allow anyone to make changes to the diagrams and keep the same style/colors. I'll add a note about the diagrams.